blog/_posts/Hola, Amigos.md

```diff
@@ -0,0 +1,97 @@
+---
+**title:** We solemnly swear we’re upto no good!
+**layout:** post
+**created_at:** Mon Jul 09 2018
+**permalink:** blog/2018-07-09-team-sectumsempra
+**current:** blog
+**author:** Avneet Kaur
+**twitter:** TeamSectumsem18
+**categories:**  student-posts
+---
+
+# "We solemnly swear we’re upto no good."
+
+## The Beginning: How delayed rewards brought us extra happiness!
+After a thorough application process, a long wait and an interview, we thought we might just be able to make it through, and were eagerly waiting for that one “Welcome to RGSoC” email. In time, an email popped up in our feed which, lured us into thinking we emerged victoriously,  but Alas! When the first round of scholars was announced, we were heart-broken that we didn't make the cut. However, the same email also stated that we were waitlisted candidates, giving us a glimmer of hope. With our fingers crossed, we wished for the best! Almost after another month long wait, we were no less than flabbergasted, when we saw an email from RGSoC, quietly sitting in our inbox, giving us the much awaited news - We were officially in! And we couldn’t contain our excitement thereafter!
+This reminds us of the famous quote from the *Harry Potter series*,
+
+> *“Things we lose have a way of coming back to us in the end, if not always in the way we expect.”  ― Luna Lovegood, Harry Potter and the Order of the Phoenix."*
+
+And hence began our official journey with RGSoC, and while currently working on our first task, all we wish for and would work towards is making an impact, and let our work speak for ourselves. We are reminded of a few lines, which we truly resonate with: 
+ 
+>        “Lives of great men, all remind us,
+>        That we can make our lives sublime,
+>        And departing, leave behind, us, 
+>        Footprints, on the sands of time”  
+>                    - H.W. Longfellow 
+ Footprints, is what we wish to leave behind. :D 
+ 
+ ![Alt Text](https://github.com/avneet14027/images/blob/master/2018-07-09-team-sectumsempra-blog-teampic.jpeg)
```

Comment:
Please add image description as well. For example, who is in the image etc. And then in brackets, you can write about image source. As the blog post is from a team and not a single person, 'self' is a confusing word as a source. You need to specify the name or device etc.